### ISSUE TITLE
Missing aria-label to indicate state of lights, buttons, switches etc on Lovelace dashboards

### ISSUE THREAD

**Issue by digitaldarragh** [home-assistant/frontend#11545]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

This is an example of the problem.   But you will find it on all elements where the state of the entity is indicated by changing it's icon. 
If a light or a button is added to a Lovelace dashboard, the state of that entity is not avilable to a screen reader.  This is because the graphic changes but for logical reasons, the text of that entity displayed on the screen doesn't change.  
To work around this, I suggest that you add an Aria label to these items.  This  aria label will contain the label displayed to users on screen with the state of that entity. 

### Describe the behavior you expected

The entty state should be reported to screen readers

### Steps to reproduce the issue

1.  Use a screen reader such as Jaws, NVDA, VoiceOver or Orca
2. Open a browser
3. Navigate to HomeAssistant
4. Check some of your lights or buttons. 
5. If the item on your Dashboard doesn't show ou some text to say that it's on or off, it's not possible to determine this using a screen reader. 

### What version of Home Assistant Core has the issue?

2022.2.0

### In which browser are you experiencing the issue with?

All

### Which operating system are you using to run this browser?

All
